hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarboniteWorkerActor.scala

```diff
@@ -30,7 +31,7 @@ class CarboniteWorkerActor(carboniterConfig: HybridCarboniteConfig,
 
   val backOff = SimpleExponentialBackoff(
     initialInterval = 5.seconds,
-    maxInterval = 5.minutes,
+    maxInterval = 15 seconds,
```

Comment:
This seems like a stowaway change?

Reply:
it is, it's the equivalent of my other PR which is now merged